src/log.js

```diff
@@ -124,7 +124,7 @@ const externalMessagesSimpleTableUrl = () =>
  * @return {string}
  */
 const messageArgToEncodedComponent = arg =>
-  encodeURIComponent(elementStringOrPassthru(arg).toString());
+  encodeURIComponent(String(elementStringOrPassthru(arg)));
```

Comment:
could we do the string coercion in `elementStringOrPassthru` though?

Reply:
It's used for `stringOrElementString` which has a different type signature. It will cast but not coerce.